make_data.py

```diff
@@ -125,62 +251,45 @@ def make_image(data, image_path, ratio:float=1.0):
     return blank
 
 
-with open(flags.labels_txt, "w") as wf:
-    image_num = 0
-    while image_num < flags.images_num:
-        image_path = os.path.realpath(os.path.join(flags.images_path, "%06d.jpg" % (image_num + 1)))
-        annotation = image_path
-        blanks = np.ones(shape=[SIZE, SIZE, 3]) * 255
-        bboxes = [[0, 0, 1, 1]]
-        labels = [0]
-        data = [blanks, bboxes, labels]
-        bboxes_num = 0
-
-        # small object
-        ratios = [0.5, 0.8]
-        # 2022-08-17 - Switched to seeded randomizer
-        N = randomizer.randint(0, flags.small)
-
-        if N != 0: bboxes_num += 1
-        for _ in range(N):
-            # 2022-08-17 - Switched to seeded randomizer
-            ratio = randomizer.choice(ratios)
-            idx = randomizer.randint(0, 54999)
-
-            data[0] = make_image(data, image_paths[idx], ratio)
-
-        # medium object
-        ratios = [1., 1.5, 2.]
-        # 2022-08-17 - Switched to seeded randomizer
-        N = randomizer.randint(0, flags.medium)
-        if N != 0: bboxes_num += 1
-        for _ in range(N):
-            # 2022-08-17 - Switched to seeded randomizer
-            ratio = randomizer.choice(ratios)
-            idx = randomizer.randint(0, 54999)
-            data[0] = make_image(data, image_paths[idx], ratio)
-
-        # big object
-        ratios = [3., 4.]
-        # 2022-08-17 - Switched to seeded randomizer
-        N = randomizer.randint(0, flags.big)
-        if N != 0: bboxes_num += 1
-        for _ in range(N):
-            # 2022-08-17 - Switched to seeded randomizer
-            ratio = randomizer.choice(ratios)
-            idx = randomizer.randint(0, 54999)
-            data[0] = make_image(data, image_paths[idx], ratio)
-
-        if bboxes_num == 0: continue
-        cv2.imwrite(image_path, data[0])
-        for i in range(len(labels)):
-            if i == 0: continue
-            xmin = str(bboxes[i][0])
-            ymin = str(bboxes[i][1])
-            xmax = str(bboxes[i][2])
-            ymax = str(bboxes[i][3])
-            class_ind = str(labels[i])
-            annotation += ' ' + ','.join([xmin, ymin, xmax, ymax, str(class_ind)])
-        image_num += 1
-        print("=> %s" % annotation)
-        wf.write(annotation + "\n")
+# 2022-09-15 - sei-hmmoore Wrapped main logic into main() function
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--images_num", type=int, default=1000)
+    parser.add_argument("--image_size", type=int, default=416)
+    parser.add_argument("--images_path", type=str, default="./yymnist/Images/")
+    parser.add_argument("--labels_txt", type=str, default="./yymnist/labels.txt")
+    # sei-hmmoore 2022-09-15
+    parser.add_argument("--coco_json", type=str, default="./yymnist/Images.json")
```

Comment:
Prefer lowercase names like `images.json`.  Also, can we add help with at least says "Defaults to...".